sunshine/confighttp.cpp

```diff
@@ -221,6 +221,27 @@ void getTroubleshootingPage(resp_https_t response, req_https_t request) {
   response->write(header + content);
 }
 
+void getFaviconImage(resp_https_t response, req_https_t request) {
+  print_req(request);
+
+  std::string content = read_file(WEB_DIR "images/favicon.ico");
+  response->write(content);
+}
+
+void getSunshineLogoImage(resp_https_t response, req_https_t request) {
+  print_req(request);
+
+  std::string content = read_file(WEB_DIR "images/logo-sunshine-45.png");
+  response->write(content);
+}
```

Comment:
Images not being returned correctly (obviously it's trying to make it a string). How to make it return an image?

Also, can we just serve everything in the images directory?

Reply:
Thank you!f Still getting up to speed on c++, but this all makes sense. Much appreciated!